extension/rikaicontent.ts

```diff
@@ -1286,3 +1297,5 @@ chrome.runtime.onMessage.addListener((request) => {
 
 // When a page first loads, checks to see if it should enable script
 chrome.runtime.sendMessage({ type: 'enable?' });
+
+export { RcxContent as TestOnlyRcxContent };
```

Comment:
I set it up so this gets stripped in prod (since it actually breaks prod); but I wonder if I should add a comment. (Maybe in the future ic could use something generic like `// TESTONLY: <desc>` which will cause the next line to be stripped.

Reply:
Comments are hard to use actually because esbuild strips all of them. So far going forward it will proably work just to use TestOnly prefix in a identifier on the line somewhere.